RELEASE_NOTES.md

```diff
@@ -1,5 +1,77 @@
 <!-- markdown-link-check-disable -->
 
+# 0.65.0
+
+## New Features and Improvements
+
+### Lightning AI Studio Orchestrator
```

Comment:
This should not be in here IMO, this should not be an official feature until the orchestrator is stable. And it's far from that for now.

Reply:
Ok, I will silent it down